applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -414,6 +414,16 @@ void NodalConcentratedElement::Initialize()
             mELementalFlags.Set(NodalConcentratedElement::COMPUTE_ROTATIONAL_DAMPING_RATIO, true);
         } else
             mELementalFlags.Set(NodalConcentratedElement::COMPUTE_ROTATIONAL_DAMPING_RATIO, false);
+
+        // We check the initial displacement
+        if (!this->Has(INITIAL_DISPLACEMENT)) {
```

Comment:
Maybe is better to check that the variable exists on running time, otherwise we will spend that precious memory in store zeros

Reply:
So it should also not be predefined in e.g. lines 375 and 384? 
For one of my tests, I did not include the nodal stiffness in my properties, so I needed the code extension of my commit.

Reply:
Wait, I will chenge it